schedule.go

```diff
@@ -27,16 +27,17 @@ type PaginatedSchedulesResponse struct {
 }
 
 type Schedule struct {
-	ID               string         `json:"id"`
-	TeamId           string         `json:"team_id"`
-	Type             string         `json:"type"`
-	OnCallNow        []string       `json:"on_call_now"`
-	Name             string         `json:"name"`
-	ICalUrlPrimary   *string        `json:"ical_url_primary"`
-	ICalUrlOverrides *string        `json:"ical_url_overrides"`
-	TimeZone         string         `json:"time_zone"`
-	Slack            *SlackSchedule `json:"slack"`
-	Shifts           *[]string      `json:"shifts"`
+	ID                 string         `json:"id"`
+	TeamId             string         `json:"team_id"`
+	Type               string         `json:"type"`
+	OnCallNow          []string       `json:"on_call_now"`
+	Name               string         `json:"name"`
+	ICalUrlPrimary     *string        `json:"ical_url_primary"`
+	ICalUrlOverrides   *string        `json:"ical_url_overrides"`
+	EnableWebOverrides bool           `json:"enable_web_overrides"`
+	TimeZone           string         `json:"time_zone"`
+	Slack              *SlackSchedule `json:"slack"`
+	Shifts             *[]string      `json:"shifts"`
```

Comment:
OnCall api returns null if schedule has no shifts? Double-checking if shifts should be pointer. Same question about Create/Update options.

Reply:
From what I can see, `shifts` will always be a list (empty if there are no shifts related to the schedule). And it should be a list when passing it on creation/update (being the param optional).